docs/source/configuration.rst

```diff
@@ -325,9 +333,15 @@ Options reference
 
    * - ::
 
-        deployment-dir = PATH
+        artifacts-dir = PATH
+
+     - Path to the deployment artifact directory as it can be seen `here <https://github.com/beamer-bridge/beamer/tree/main/deployments/artifacts>`_.
+
+   * - ::
+
+        abi-dir = PATH
 
-     - Path to the deployment directory as it can be seen `here <https://github.com/beamer-bridge/beamer/tree/07d66e0bb8c76bb1ff219e24e34e1c24ee7890c6/deployments>`_.
+     - Path to the contract abi files directory.
```

Comment:
This is not consistent within the docs, but also not with the change that was made in `call_contracts` script (the help text). I think we should use the same wording both in the docs and in the script help text i.e. we should make this match the wording from the `call_contracts` script help text.